using_images/xpaas_images/eap.adoc

```diff
@@ -9,7 +9,94 @@
++
+----
+oc exec p <pod> -c <container> - /opt/eap/bin/jboss-cli.sh
+----
++
+[WARNING]
+Any configuration changes made using the JBoss EAP Management CLI on a running container is not persistent. 
+* Domain mode is not supported. OpenShift controls the creation and distribution of applications in the containers.
+* There is no HornetQ messaging subsystem. The JBoss A-MQ xPaaS image can be used for scenarios that require messaging. 
+
+[[Managing-OpenShift-JBoss-EAP-xPaaS-Images]]
+=== Managing OpenShift JBoss EAP xPaaS Images
+
+A major difference in managing an OpenShift JBoss EAP xPaaS image is that there is no Management Console exposed for the JBoss EAP installation inside the image.
+
+As containers are not persistent, the JBoss EAP Management Console is not included. However, the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*) is still accessible from within the container for troubleshooting purposes, however any configuration changes made on a running OpenShift container will be lost when the container is destroyed.
+
+link:#Making-Configuration-Changes-EAP[Making configuration changes to the JBoss EAP instance inside the JBoss EAP xPaaS image] is different to the process you may be used to for a regular release of JBoss EAP. 
+
+=== Unsupported Configurations
+
+The following is a list of unsupported configurations specific to the JBoss EAP xPaaS image:
+
+* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead. 
+// This is based on https://issues.jboss.org/browse/CLOUD-56
+
+== Installing the JBoss EAP xPaaS Image Streams and Application Templates
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and source-to-image (S2I) application templates.
+
+. Clone the Application Template repository:
++
+----
+# git clone https://github.com/jboss-openshift/application-templates
+----
+. Create the image streams and S2I templates for your OpenShift project:
++
+----
+# oc create -f application-templates/jboss-image-streams.json -n <project>
+# oc create -f application-templates/eap/ *.json -n <project>
+----
+
+You will now have the JBoss EAP xPaaS images streams and S2I application templates available to use in your OpenShift project.
+
+[[Making-Configuration-Changes-EAP]]
+== Running and Configuring the JBoss EAP xPaaS Image
+
+There are a few methods of making changes to the JBoss EAP configuration used in the JBoss EAP xPaaS image.
+
+=== Using Environment Variables and Application Template Parameters
+
+The recommended method to run and configure the JBoss EAP xPaaS Image is to use the OpenShift S2I process together with the application template parameters and environment variables.
```

Comment:
@lucascosti There's a thought against using "recommended" in the upstream docs. For this, I'd give reasons why this is the recommended method, without actually using the golden word.

So, why is this recommended?